helpers/azure/api.js

```diff
@@ -910,6 +915,13 @@ var postcalls = {
             properties: ['id'],
             url: 'https://management.azure.com/{id}/configurations?api-version=2017-12-01'
         }
+    },
+    mediaServices: {
```

Comment:
we can simply use listAll 
![Uploading Screenshot 2023-12-04 at 3.38.32 PM.png…]()